### ISSUE TITLE
European Portuguese Translation Project Menu Image

### ISSUE THREAD

**Issue by bascogamer** [elishacloud/Silent-Hill-2-Enhancements#822]
While doing my European Portuguese translation project for Silent Hill 2, I encountered a problem when translating the main menu. 

I converted the High Quality Texture to DSS and edited it to be in Portuguese, but then I was unable to re-convert it to the ".tex" format. 

Would any kind of assistence be possible? 

What do I need to proceed?

**Comment by Polymega** [comment]
Hi there,

Very cool to hear you making a European Portuguese translation pack! I can help you with translating the main menu texture. I'll need the following translated to create this:

```
LOAD
CONTINUE
NEW GAME
OPTIONS
MOVIES
EXIT
ACTION LEVEL
RIDDLE LEVEL
HARD
NORMAL
EASY
BEGINNER
Main Scenario: LETTER FROM SILENT HEAVEN
Sub Scenario: BORN FROM A WISH
```

Thanks

**Comment by bascogamer** [comment]
Hey, thanks for answering and for helping.

Here goes the translation:


```
CARREGAR
CONTINUAR
NOVO JOGO
OPÇÕES
CENAS
SAIR
NÍVEL DA AÇÃO
NÍVEL DOS ENIGMAS 
DIFÍCIL
NORMAL
FÁCIL
PRINCIPIANTE
História Principal: CARTA DE UM PARAÍSO PLÁCIDO
História Secundária: NASCIDA DE UM SONHO
``` 

Again, thank you very much for helping.

**Comment by lucazandrade** [comment]
It's great to know that a translation into European Portuguese is being made, I'm very happy. Anything if you need help with the texts, or information regarding the peculiarities and standardization, textual characteristics, I am available to help!!

**Comment by bascogamer** [comment]
> 

Yeah, I searched if it was possible to mod, and it was preety much possible, so I started to work on it. As far as progress goes I would say it is going pretty well, aside from the main menu texture issue. I'll try and do a dub for the game too since it worked well on the test I made.

**Comment by Polymega** [comment]
Here you go: [start01x.zip](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/12177087/start01x.zip)

Note that @lucazandrade did an _outstanding_  job creating a Brazilian Portuguese translation pack for the project. So if you have any questions whatsoever and he has the time to help, be sure to ask Lucaz!

Also, we'd be interested in hosting your European Translation pack on our [project website](https://enhanced.townofsilenthill.com/SH2/languagepacks.htm) once its complete. Just note we could only host translated `.mes` files (along with the translated main menu); we wouldn't be able to host translated dubbing/images.

**Comment by bascogamer** [comment]
> 

Thank you very much! I would love for you to host the translation, even it's just the text and menu. 

Again, thanks a lot!
